edalize/tools/vpr.py

```diff
@@ -90,27 +99,68 @@ def configure(self, edam):
 
         commands = EdaCommands()
 
-        depends = self.name + ".blif"
+        depends = netlist_file
         targets = self.name + ".net"
-        command = ["vpr", arch_xml, self.name + ".blif", "--pack"]
+        command = ["vpr", arch_xml, netlist_file, "--pack"]
         command += sdc_opts + vpr_options
         commands.add(command, [targets], [depends])
 
-        depends = self.name + ".net"
+        # Run generate constraints script of flag set to true
+        if self.tool_options.get("gen_constraints", False):
```

Comment:
`if self.tool_options.get("gen_constraints"):` is enough since `get` will return `None` which is a false value. (No big deal though. Just FYI). Also `of` should be `if` in the comment above

Reply:
Fixed both in 6c43fcf